### DISCUSSION TITLE
EDC 0.7.1 →  `AmazonS3-PUSH` and `AzureStorage-PUSH` →  How to deactivate?

### DISCUSSION THREAD

**Discussion by jerzy-wachowiak** [eclipse-tractusx/tractusx-edc#1335] [Q&A]
After catalog query I get
```json
{
    " @type": "dcat:Distribution",
    "dct:format": {
        "@id": "AzureStorage-PUSH"
    },
    "dcat:accessService": {
        "@id": "c3bbc525-60fe-4202-a71a-ab05c680e4c3",
        "@type": "dcat:DataService",
        "dct:terms": "connector",
        "dct:endpointUrl": "https://bpnl00000007rnyv.edc.supplyon.com/api/v1/dsp"
}
```
and
```json
{
    "@type": "dcat:Distribution",
    "dct:format": {
        "@id": "AmazonS3-PUSH"
    },
    "dcat:accessService": {
        "@id": "c3bbc525-60fe-4202-a71a-ab05c680e4c3",
        "@type": "dcat:DataService",
        "dct:terms": "connector",
        "dct:endpointUrl": "https://bpnl00000007rnyv.edc.supplyon.com/api/v1/dsp"
    }
}
```

How can I deactivate `AmazonS3-PUSH` and `AzureStorage-PUSH` ? We do not offer this services...

**Comment by paullatzelsperger** [comment]
as per the [documentation](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/docs/usage/management-api-walkthrough/04_catalog.md#returned-payload), this depends on your data plane configuration. 
This is configured in the Helm chart with some [defaults](https://github.com/eclipse-tractusx/tractusx-edc/blob/c34b9c9eb8bf7e5085e42608eb14d95e38b2fb44/charts/tractusx-connector/templates/deployment-controlplane.yaml#L309-L315).

You can override these environment variables by configuring `controlplane.env` in your chart.

**Reply by jerzy-wachowiak** [reply]
So I can remove them and it does not harm to the EDC, correct?

**Reply by paullatzelsperger** [reply]
you can change the environment variables omitting the `AmazonS3-PUSH` etc. and that will prevent those assets from being displayed in the catalog as "transferrable via S3"

**Comment by jerzy-wachowiak** [comment]
BTW: `AmazonS3-PUSH` and `AzureStorage-PUSH` are not part of the https://docs.internationaldataspaces.org/ids-knowledgebase/v/dataspace-protocol

**Reply by paullatzelsperger** [reply]
DSP only relates to the communication between two EDC _control planes_. Naturally, it cannot make any assumptions or assertions regarding implementation details of the data transmission protocol.

**Reply by jerzy-wachowiak** [reply]
Does it mean, that there is no standard for data transmission protocol?

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
